src/main.cpp

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Sean Cummings
+#include <string>
+#include <memory>
+#include <iostream>
+#include "camera.h"
+#include "mock.h"
+#include "ucam.h"
+
+typedef struct {
+    bool takePicture;
+    char imgLogcation[50];
+} CameraType;
```

Comment:
Is `CameraType` still needed? I don't see a reference to this anywhere.

Reply:
@JoshuaHassler @sjc625 ?

Reply:
It will probably be when I integrate it with octopos